lib/create_response.js

```diff
@@ -0,0 +1,60 @@
+const { IncomingHttpHeaders, IncomingMessage } = require('http')
+
+/**
+ * Creates a Fetch API `Response` instance from the given
+ * `http.IncomingMessage` instance.
+ * copied from: https://github.com/mswjs/interceptors/blob/04152ed914f8041272b6e92ed374216b8177e1b2/src/interceptors/ClientRequest/utils/createResponse.ts#L8
+ */
+
+/**
+ * @param {IncomingMessage} message 
+ */
+function createResponse(message) {
+  const readable = new ReadableStream({
```

Comment:
Looks like the version of Node Nock uses doesn't have the `ReadableStream` global. I'd double-check if this isn't the linter's problem as the first measure.

Reply:
Sometimes the linter doesn't play well with relatively newly added globals. This can be modified in the linter's `globals` key, if talking about ESLint.

Reply:
Thanks! 
I think we can fix this after we will solve the `got` error. most of Nock's tests are based on it and I'm eager to solve it and run all tests to find cases that we haven't covered yet.

Reply:
I will try to look into the `got` issue this week based on my availability. Right now, the best next step is to reliably reproduce it in an automated test. I've already written one (see https://github.com/mswjs/interceptors/pull/432?notification_referrer_id=NT_kwDOAOSmz7M3NzYwMzQ3MDA1OjE0OTg0OTEx&notifications_query=is%3Aunread#issuecomment-1724139617) but you've provided some input that I need to reflect in the test.

Reply:
I've fixed the `got `issue, would appreciate a quick review on the pull request!